samples/swift/uidemo.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,575 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
+	classes = {
+	};
+	objectVersion = 46;
+	objects = {
+
+/* Begin PBXBuildFile section */
+		6BEBFBD2CFCDA8A7918028C1 /* Pods_uidemoTests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 1A128E4DD412913B07E20C37 /* Pods_uidemoTests.framework */; };
+		8DABC9891D3D82D600453807 /* AppDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC9881D3D82D600453807 /* AppDelegate.swift */; };
+		8DABC98B1D3D82D600453807 /* MenuViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC98A1D3D82D600453807 /* MenuViewController.swift */; };
+		8DABC98E1D3D82D600453807 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 8DABC98C1D3D82D600453807 /* Main.storyboard */; };
+		8DABC9901D3D82D600453807 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8DABC98F1D3D82D600453807 /* Assets.xcassets */; };
+		8DABC9931D3D82D600453807 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 8DABC9911D3D82D600453807 /* LaunchScreen.storyboard */; };
+		8DABC99E1D3D82D600453807 /* uidemoTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC99D1D3D82D600453807 /* uidemoTests.swift */; };
+		8DABC9A91D3D872C00453807 /* Sample.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC9A81D3D872C00453807 /* Sample.swift */; };
+		8DABC9AB1D3D947300453807 /* SampleCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC9AA1D3D947300453807 /* SampleCell.swift */; };
+		8DABC9AD1D3D9EAF00453807 /* ChatViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8DABC9AC1D3D9EAF00453807 /* ChatViewController.swift */; };
+		8DABC9AF1D3DA30D00453807 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 8DABC9AE1D3DA30D00453807 /* GoogleService-Info.plist */; };
```

Comment:
I think we're talking about different errors. When I try running this with the reference but without the plist, I get a compiler error saying it can't find the file. Dragging the plist into Xcode then produces two valid references pointing to the same file (ugly, would like to avoid), but the project does build and run fine. Removing the reference gives me a runtime error on app launch that says it can't find the plist file. In this case I still would prefer the former, since the red broken reference in Xcode gives some more context as to where the project expects the plist file to be; the runtime error says it can't find it, but doesn't mention where it looked.

The distinction is small imo and not a killer either way and seems largely dependent on Xcode behavior (which I'm willing to disregard for the sake of consistency), but I wanted to disambiguate stuff before making changes.